style/css/printable.css

```diff
@@ -0,0 +1,9 @@
+/*
+ * This is a CSS included in the printable version of docs, it's used to mask out large images etc
+ */
```

Comment:
not sure this does anything to large images yet but makes sense that would be the intention (no need to change if it is)